.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "decco"]
+	path = decco
+	url = git@github.com:input-output-hk/decco.git
```

Comment:
Why aren't we using the `https://github.com/input-output-hk/decco.git` URL?

Reply:
I didn't realize this was the recommended approach. Changed to https.